blas/unit_test/Test_Blas1_nrm2w.hpp

```diff
@@ -22,28 +22,32 @@
 namespace Test {
 template <class ViewTypeA, class Device>
 void impl_test_nrm2w(int N) {
-  typedef typename ViewTypeA::value_type ScalarA;
-  typedef Kokkos::ArithTraits<ScalarA> AT;
+  using ScalarA    = typename ViewTypeA::value_type;
+  using AT         = Kokkos::ArithTraits<ScalarA>;
+  using MagnitudeA = typename AT::mag_type;
 
   ViewTypeA a("A", N);
   ViewTypeA w("W", N);
 
   typename ViewTypeA::HostMirror h_a = Kokkos::create_mirror_view(a);
   typename ViewTypeA::HostMirror h_w = Kokkos::create_mirror_view(w);
 
+  const MagnitudeA max_val = 10;
+  const MagnitudeA eps     = AT::epsilon();
+  const MagnitudeA max_error =
+      max_val * std::sqrt(static_cast<MagnitudeA>(N)) * eps;
```

Comment:
`N * eps` makes sense to me WRT to usage in `EXPECTED_NEAR_KK`. Why is that multiplied by `max_val` too?

Reply:
Think about cancellation errors: `y = 1002 - 1001 = 1` you would think the accuracy is `eps` because of the value in `y` but actually it is `1000*eps`